platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/SystemPropertiesAdmin.java

```diff
@@ -122,11 +120,6 @@ public SystemPropertiesAdmin(GuestClaimsHandlerExt guestClaimsHandlerExt)
     super(SystemPropertiesAdminMBean.class);
     this.guestClaimsHandlerExt = guestClaimsHandlerExt;
     configureMBean();
-    try {
-      systemDotProperties = new Properties(systemPropertiesFile);
```

Comment:
✏️ I think we should add a unit test like

1. instantiate this class
2. override one value in system.properties and set the system property (might want to try both a property than can be changed in the installer page and one that isn't, like the solr property)
3. `readSystemProperties`
4. `writeSystemProperties`